src/configure/configure.ts

```diff
@@ -154,6 +157,23 @@ class Orchestrator {
         }
     }
 
+    private async getGithubPatToken(): Promise<void> {
+        const redirectHelper = new RedirectLinkHelper();
+        await redirectHelper.loadAll();
+        if (this.inputs.sourceRepository.repositoryProvider === "github" || redirectHelper.repoAnalysisServiceFramework === ServiceFramework.Moda) {
```

Comment:
Shouldn't we get pat irrespective of whether it is moda or vssf service? Are you getting pat token in case of vssf service somewhere else?

Reply:
In case of vssf service, we need to get PAT only for the flows when repository is of type github. For Azure Repo  + Azure Pipelines flow, we don't need